charts/langsmith/templates/_helpers.tpl

```diff
@@ -210,10 +209,36 @@ Template containing common environment variables that are used by several servic
 - name: TRACE_TIER_TTL_DURATION_SEC_MAP
   value: "{ \"longlived\": {{ .Values.config.ttl.ttl_period_seconds.longlived }}, \"shortlived\": {{ .Values.config.ttl.ttl_period_seconds.shortlived }} }"
 {{- end }}
+- name: FF_WORKSPACE_SCOPE_ORG_INVITES_ENABLED
+  value: {{ .Values.config.workspaceScopeOrgInvitesEnabled | quote }}
 {{- if .Values.config.orgCreationDisabled }}
 - name: FF_ORG_CREATION_DISABLED
   value: {{ .Values.config.orgCreationDisabled | quote }}
 {{- end }}
+{{- if .Values.config.blobStorage.enabled }}
```

Comment:
is it assumed that this is AWS and not another cloud provider for now? or the `S3` env vars will get used for all providers?